larq/quantizers.py

```diff
@@ -164,6 +165,45 @@ def non_trainable_weights(self):
         return []
 
 
+@utils.register_keras_custom_object
+class NoOpQuantizer(BaseQuantizer):
+    r"""Instantiates a serializable no-op quantizer.
```

Comment:
Why the `r`? You're already escaping `\`s right?

Reply:
I copied this from the other quantizers, so I guess it is needed to correctly render MatJax.